### PR TITLE
fix(dialogs): make `scale` size prop take effect only on the horizontal constraints

### PR DESCRIPTION
**Pull request by kark**
#### Summary

|  Before | After  |
|---|---|
|  Screenshot 2024-01-23 at 08 45 45 |  Screenshot 2024-01-23 at 08 48 14 |

### CODE REVIEW

File: website-custom-views/src/content/api-reference/commercetools-frontend-application-components.mdx [hunk 1]
```diff
@@ -1130,7 +1130,7 @@ const ChannelsInfo = () => {
...
 | `children`          | `node`     |    ✅    | -                 | -       | Content rendered within the dialog. If the content is long in height (depending on the screen size) a scrollbar will appear.                                                                                                                                                                                    |
-| `size`              | `string`   |    -     | `m`, `l`, `scale` | `l`     | Horizontal width of the dialog card. If `scale` is used, the dialog size expands to the viewport size, with some margin.                                                                                                                                                                                                                                                                            |
```
Comment (emmenko) [inline website-custom-views/src/content/api-reference/commercetools-frontend-application-components.mdx:1133]: I think the comment is still valid regarding `scale`, we don't need to change that.

**Review by emmenko — COMMENTED** [review]
Thanks for the fix! 🙌

File: website-custom-views/src/content/api-reference/commercetools-frontend-application-components.mdx (hunk 1 above)
Comment (kark) [reply website-custom-views/src/content/api-reference/commercetools-frontend-application-components.mdx]: Yes, I believe so, thanks

**Review by CarlosCortizasCT — APPROVED** [review]
Thanks a lot! 🙇

**Review by emmenko — APPROVED** [review]
💯

**Review by Rhotimee — APPROVED** [review]
💯

**Pull request merged**